rpcs3/Loader/PSF.cpp

```diff
@@ -251,6 +251,14 @@ namespace psf
 			PSF_CHECK(false, corrupt);
 		}
 
+		const auto tid = get_string(pair.sfo, "TITLE_ID", "");
+
+		if (std::count_if(tid.begin(), tid.end(), [](char ch){ return !((ch >= 'a' && ch <= 'z') || (ch >= 'A' && ch <= 'Z') || (ch >= '0' && ch <= '9')); }) != 0)
```

Comment:
why not find_if ?

Reply:
Im lazy and it's the same performance wise if it doesn't find anything.

Reply:
fixed.